### PR TITLE
apply editor feedback to CMS comparison

### LINKED ISSUE #1204: Full audit for CMS Comparison: Drupal, Joomla, and WordPress
https://support.rackspace.com/how-to/cms-comparison-drupal-joomla-and-wordpress/

We need to do a full audit on this Cloud Sites article because it will be linked to from our new Getting Started site. The last update was in January 2016.

### PR DESCRIPTION
**Pull request by stephamon**
Fixes #1204

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-19fe54ff3c/how-to/)

File: content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
 
-There's no one-size-fits-all solution here; it depends on your goals, technical expertise, budget and what you need your site to do. For a simple blog or brochure-type site, [Wordpress](http://www.rackspace.com/cloud/sites/web-hosting/wordpress/) could be the best choice (while very friendly for non-developers, it's a flexible platform also capable of very complex sites). For a complex, highly customized site requiring scalability and complex content organization, [Drupal](http://www.rackspace.com/cloud/sites/web-hosting/drupal/) might be the best choice. For something in between that has an easier learning curve, [Joomla](http://www.rackspace.com/cloud/sites/web-hosting/joomla/) may be the answer.
+Your choice depends on your goals, technical expertise, budget and what you the site to do.
```
Comment (catlook) [inline content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md]: ... what you want the site ...

File: content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
 
-When you have questions or need help, will you be able to find it easily? With all of these systems, the answer is yes. Each has passionate, dedicated developer and user communities, making it easy to find free support directly through their websites or through other online forums or even books. In addition, paid support is readily available from third-party sources, such as consultants, developers and designers. Each of these systems shows long-term sustainability and longevity; support for them will continue to be readily available for the foreseeable future. The more time and effort you are willing and able to invest into learning a system, the more it will be able to do for you. With both Wordpress and Joomla, you can order a wide range of services and options off the menu to suit your needs; with Drupal, you'll be in the kitchen cooking up what you want for yourself, with all of the privileges of customization that entails.
+- For a simple blog or brochure-type site, [WordPress](http://www.rackspace.com/cloud/sites/web-hosting/wordpress/) could be the best choice (while very friendly for non-developers, it's a flexible platform also capable of very complex sites).
```
Comment (catlook) [inline content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md]: "flexible platform also capable of very complex sites" --- capable of creating?  managing? delivering?

File: content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md [hunk 3]
```diff
@@ -1,31 +1,37 @@
...
+- For something in between that has an easier learning curve, [Joomla](http://www.rackspace.com/cloud/sites/web-hosting/joomla/) might be the answer.
 
-See the comparison chart below for more insight into the differences in these top content management systems. Still not sure? Download each of the free platforms and do a trial run to help you decide.
+Each has passionate, dedicated developer and user communities, making it easy to find free support directly through their websites or through other online forums or even books. In addition, paid support is readily available from third-party sources, such as consultants, developers and designers. Each system shows long-term sustainability and longevity; support for them will continue to be readily available for the foreseeable future.
 
-|   | **Drupal** | **Joomla** | **Wordpress** |
+The more time and effort that you are willing and able to invest in learning a system, the more it will be able to do for you. With both WordPress and Joomla, you can order a wide range of services and options from the menu to suit your needs; with Drupal, you can customize what you need.
```
Comment (catlook) [inline content/cloud-sites/cms-comparison-drupal-joomla-and-wordpress.md]: ... suit your needs, and with Drupal ....  [this is just my pref...]

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-46b8805aaf/how-to/)

**Comment by catlook** [comment]
👍

**Pull request merged**